src/SFA.DAS.Assessors/InnerApi/Responses/StandardDetailResponse.cs

```diff
@@ -0,0 +1,60 @@
+﻿using SFA.DAS.SharedOuterApi.InnerApi.Responses;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.Assessors.InnerApi.Responses
+{
+    public class StandardDetailResponse : StandardApiResponseBase
+    {
+        public string StandardUId { get; set; }
+        public string IfateReferenceNumber { get; set; }
+        public int LarsCode { get; set; }
+        public string Status { get; set; }
+        public float? SearchScore { get; set; }
+        public string Title { get; set; }
+        public int Level { get; set; }
+        public decimal Version { get; set; }
+        public string OverviewOfRole { get; set; }
+        public string Keywords { get; set; }
+        public string Route { get; set; }
+        public string AssessmentPlanUrl { get; set; }
+        public string TrailBlazerContact { get; set; }
+        public string TypicalJobTitles { get; set; }
+        public string CoreSkillsCount { get; set; }
+        public List<string> Skills { get; set; }
+        public List<string> Knowledge { get; set; }
+        public List<string> Behaviours { get; set; }
+        public string StandardPageUrl { get; set; }
+        public string IntegratedDegree { get; set; }
+        public decimal SectorSubjectAreaTier2 { get; set; }
+        public string SectorSubjectAreaTier2Description { get; set; }
+        public bool OtherBodyApprovalRequired { get; set; }
+        public string ApprovalBody { get; set; }
+        public List<string> Duties { get; set; }
+        public bool CoreAndOptions { get; set; }
+        public string CoreDuties { get; set; }
+        public bool IntegratedApprenticeship { get; set; }
+        public List<string> Options { get; set; }
+
+        public StandardVersionDetail VersionDetail { get; set; }
+
+        public EqaProvider EqaProvider { get; set; }
+
+        protected override int GetFundingDetails(string prop)
+        {
+            if (ApprenticeshipFunding == null || ApprenticeshipFunding.Any() == false)
```

Comment:
Is this the best way to do this? Understand that if the ApprenticeshipFunding is null or doesn't have any records then we fall back to Ifate funding fields from Version Detail.  Just wondering if this would cause confusion whereby you don't know if the funding was Lars originated & ESFA reviewed vs IFate data which you could get via inspecting the version detail object on the consumer side.

I'm cautious of giving the consumer the wrong impression of where the data came from so just worth considering.

Reply:
I had raised this here, see the comment section. https://skillsfundingagency.atlassian.net/wiki/spaces/NDL/pages/2758574194/Apprenticeships+Assessment+Service+Database+-+SV#TO-BE-RoEPAO-Database-design